src/main/controllers/ApplicationDetailsController.ts

```diff
@@ -19,6 +23,16 @@ export default class ApplicationDetailsController {
       return res.redirect(ErrorPages.NOT_FOUND);
     }
 
+    try {
+      const newStatus: LinkStatus = getApplicationStatusAfterViewed(selectedApplication.value, req.session.user);
+      if (newStatus) {
+        await getCaseApi(req.session.user?.accessToken).changeApplicationStatus(req, selectedApplication, newStatus);
+        changeApplicationState(selectedApplication.value, req.session.user, newStatus);
+      }
+    } catch (error) {
+      res.redirect(ErrorPages.NOT_FOUND);
```

Comment:
error which is caught, is neither logged nor shown to screen, maybe log the error?

Reply:
Approved it not disregarding the comment from Gaurav.